common/src/main/resources/properties/map.properties.example

```diff
@@ -1,4 +1,4 @@
 token_route = https://www.arcgis.com/sharing/rest/oauth2/token
-feature_layer_route = https://services7.arcgis.com/iIw2JoTaLFMnHLgW/ArcGIS/rest/services/sftt_development_layer/FeatureServer/0/applyEdits
+feature_layer_route = https://services7.arcgis.com/iIw2JoTaLFMnHLgW/ArcGIS/rest/services/sftt_development_layer/FeatureServer/0/
```

Comment:
IMPORTANT: update local URL

Reply:
ALSO IMPORTANT: @justinkonecny we have to remember to update the value in travis

Reply:
Very important to know yeah 👍

Reply:
No, I want to limit the number of properties as much as possible to prevent unpredictable downstream effects in deployment.